index.js

```diff
@@ -23,8 +23,10 @@ try {
 module.exports = firewall;
 module.exports.forgetKnownHosts = forgetKnownHosts;
 
-function firewall(app) {
+function firewall(passedObject) {
```

Comment:
If you don't mind I would like to be pedantic here. This is a two-lines diff, but I'm going to write more than two lines 🐨 

First of all, the name `passedObject` does not look cool. It's obvious that the object is passed because this is a name of the function parameter. So, if should be just `object` at least.

May be it would be even better not to call it `object` but simply call it `app`. Then redefine this `app` below if we should.

Reply:
I don't think that renaming this entity to `app` is a good idea, because redefinition would look kinda wierd and not readable 🤔 
```
app = 'app' in app ? app.app : app;
```

So I used `devServer` as it called in [webpack-dev-server](https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md#-breaking-changes) spec, do you mind?

Reply:
It looks fine to me, thanks!